src/runtime_src/core/common/query_requests.h

```diff
@@ -375,18 +376,32 @@ struct rom_uuid : request
 {
   using result_type = std::string;
   static const key_type key = key_type::rom_uuid;
+  static const char* name() { return "uuid"; }
 
   virtual boost::any
   get(const device*) const = 0;
+
+  static result_type
+  to_string(const result_type& value)
+  {
+    return value;
+  }
```

Comment:
Hmm is this one needed?  device_query<rom_uuid>(..) return result_type which is std::string